hass_nabucasa/api.py

```diff
@@ -174,6 +175,7 @@ def _do_log_response(
             return
         isok = resp.status < 400
         target = resp.url.path if resp.url.host == self.hostname else ""
+        target += f"?{resp.url.query_string}" if resp.url.query_string else ""
```

Comment:
I do not think we want to log the entire query string, this can/will contain sensitive contents.
An alternative is to only log the keys, and if the value is a bool, that would be acceptable as well.